suites/upgrade/hammer-x/parallel/5-final-workload/blogbench.yaml

```diff
@@ -3,11 +3,12 @@ meta:
    run a cephfs stress test
    mount ceph-fuse on client.3 before running workunit
 tasks:
-- sequential:
-  - ceph-fuse:
-  - print: "**** done ceph-fuse 5-final-workload"
-  - workunit:
-      clients:
-         client.3:
-          - suites/blogbench.sh
-  - print: "**** done suites/blogbench.sh 5-final-workload"
+full_sequential:
```

Comment:
no need to change this or the other 5-final-workload file since they are not run by the parallel task